BuildScript.Nuke/BuildScript.Nuke.csproj

```
@@ -17,12 +17,18 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="CycloneDX.Utils" Version="8.0.3" />
     <PackageReference Include="Nuke.Build" Version="8.0.0" />
     <PackageReference Include="Nuke.Common" Version="8.0.0" />
+    <PackageReference Include="SbomCleaner.Library" Version="1.0.0" />
   </ItemGroup>
 
   <ItemGroup>
     <InternalsVisibleTo Include="BuildScript.Nuke.UnitTests" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="CycloneDX" Version="4.0.0" ExcludeAssets="all" />
+  </ItemGroup>
+
```

```suggestion
    <PackageReference Include="CycloneDX" Version="4.0.0" ExcludeAssets="all" />
    <PackageReference Include="CycloneDX.Utils" Version="8.0.3" />
    <PackageReference Include="Nuke.Build" Version="8.0.0" />
    <PackageReference Include="Nuke.Common" Version="8.0.0" />
    <PackageReference Include="SbomCleaner.Library" Version="1.0.0" />
  </ItemGroup>

  <ItemGroup>
    <InternalsVisibleTo Include="BuildScript.Nuke.UnitTests" />
  </ItemGroup>
```
Would this be a good idea or was the split intentional because of the ExcludeAssets attribute?

---

I think it makes sense to split the normal dependencies and the one for the Tool which is not used otherwise.

---

okay, great. please add a comment inside the second itemgroup so we know why it's done.